src/java_bytecode/java_entry_point.cpp

```diff
@@ -485,6 +485,49 @@ bool java_entry_point(
     pointer_type_selector);
 }
 
+/// Creates the initialize methods again taking account of symbols added to the
+/// symbol table during instantionation of lazy methods since they were first
+/// created,
+/// \param symbol_table: global symbol table containing symbols to initialize
+/// \param main_class: the class containing the "main" entry point
+/// \param message_handler: message_handlert for logging
+/// \param assume_init_pointers_not_null: specifies behaviour for
+/// java_static_lifetime_init
+/// \param object_factory_parameters: specifies behaviour for
+/// java_static_lifetime_init
+/// \param pointer_type_selector: specifies behaviour for
+/// java_static_lifetime_init
+bool recreate_initialize(
+  symbol_table_baset &symbol_table,
+  const irep_idt &main_class,
+  message_handlert &message_handler,
+  bool assume_init_pointers_not_null,
+  const object_factory_parameterst &object_factory_parameters,
+  const select_pointer_typet &pointer_type_selector)
+{
+  messaget message(message_handler);
+  main_function_resultt res=
+    get_main_symbol(symbol_table, main_class, message_handler);
+  if(res.status!=main_function_resultt::Success)
```

Comment:
Not your bug, but I'm surprised that CamelCase made it into the code base.

Reply:
Can't see the line you're referring to.

Reply:
I'm referring to the use of `Success`, where I would have expected `SUCCESS`.

Reply:
This will be a separate PR after this one is merged.

Reply:
Thanks!